config/routes.rb

```diff
@@ -278,6 +262,9 @@ def not_implemented
     end
   end
 
-  match '*path', via: :all, to: not_implemented
-
+  if Feature.active?(:redirects)
+    match '*path', via: :all, to: 'catchall#not_found'
```

Comment:
I've re-added this line https://github.com/moneyadviceservice/frontend/pull/1186/files#diff-21497849d8f00507c9c8dcaf6288b136R265 does that resolve your issue for old tools?